scripts/just.config.ts

```diff
@@ -90,7 +90,7 @@ export function preset() {
   task('ts', () => {
     return series(
       'ts:compile',
-      condition('copy-compiled', () => isConvergedPackage({ projectType: 'library' })),
+      'copy-compiled',
```

Comment:
build pipeline is now driven by ts-config as the main source of truth which enables usage cross version (v8,9) without need of creating config discrepancies